### PR TITLE
Updated release process and maintainers docs with Triage role.

### PR DESCRIPTION
**Pull request by bwplotka**
Added @adrien-f  and @FUSAKLA as official triage persons! Thanks guys!

Also put @GiedriusS as next release shepherd as agreed offline.

Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
Want to make `Maintainers.md`  available on website as well [in progress]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -18,6 +18,69 @@ Feel free to contact us (preferably on Slack) anytime for feedback, questions or
...
+|-----------------------|--------------------------|------------------------------------------------------------|
+| Adrien Fillon         | `@Adrien F`              | [@adrien-f](https://github.com/adrien-f)                   |
+| Martin Chodur         | `@FUSAKLA`               | [@fusakla](https://github.com/fusakla)                     |
+
+Please reach any of the maintainer on slack or email if you want to help as well.
+
+### Triage labels
+
+To improve navigating through issues and PRs we introduce various [labels](https://github.com/improbable-eng/thanos/issues/labels). Part of triaging process for Triage and Maintainers
+is to adjust those labels if needed manual. Some `Prow` can automate a portion of this in future.
```
Comment (povilasv) [inline MAINTAINERS.md]:
```suggestion
is to adjust those labels if needed manual. `Prow` system can automate a portion of this in future.
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -18,6 +18,69 @@ Feel free to contact us (preferably on Slack) anytime for feedback, questions or
...
+* `proposal` label indicates an issue / PR with official proposal.
+* `duplicate`
+* `feature request / improvement`
+* `bug` and `bugfix`
+
+#### State
+
+Github shows some basic states, but sometimes PR is stale due to requested changes, but it's not clear from first glance.
+That's why `state: changes-requested` is helpful on those.
+
```
Comment (povilasv) [inline MAINTAINERS.md]: remove the extra blank line

File: docs/release-process.md [hunk 3]
```diff
@@ -7,33 +7,43 @@ slug: /release-process.md
...
+# For maintainers: Cutting individual release 
+
 Process of releasing a *minor* Thanos version:
 1. Release `v<major>.<minor+1>.0-rc.0`
 1. If after 3 work days there is no major bug, release `v<major>.<minor>.0`
 1. If within 3 work days there is major bug, let's triage it to fix it and then release `v<major>.<minor>.0-rc.++` Go to step 2.
 1. Do patch release if needed for any bugs afterwards. Use same `release-xxx` branch.
 
-### How to release "a version"
+## How to release "a version"
```
Comment (adrien-f) [inline docs/release-process.md]: Why in quote ?

Comment (bwplotka) [reply docs/release-process.md]: no idea ;p

**Comment by bwplotka** [comment]
Addressed all comments

**Pull request merged**